statsd/options.go

```diff
@@ -212,10 +212,10 @@ func WithSenderQueueSize(senderQueueSize int) Option {
 	}
 }
 
-// WithWriteTimeoutUDS sets the WriteTimeoutUDS option.
-func WithWriteTimeoutUDS(writeTimeoutUDS time.Duration) Option {
+// WithWriteTimeout sets the WriteTimeout option.
```

Comment:
nit: This comment could be a bit better but I know it's there from the old code

Reply:
I have another PR rewriting the doc